packages/algjulia-interop/src/decapodes-service/analysis/initial_conditions.jl

```diff
@@ -84,7 +79,7 @@ function initial_conditions(ics::Dict{Symbol,<:InitialConditions}, geometry::Geo
 end
 
 function initial_conditions(x::InitialConditions, args...)
-    throw(ImplError("These initial conditions ($(x)) are"))
+    throw(ImplError("These initial conditions ($(x)) are")) # TODO
```

Comment:
Lingering TODO: can we just delete this method? IMO, in Julia it's usually better to not define a method at all than to define and throw a not-implemented error.